test/Conversion/SCFToCalyx/convert_simple.mlir

```diff
@@ -257,3 +257,28 @@ module {
     return %1 : f32
   }
 }
+
+// -----
+
+// Test floating point mul
+
+// CHECK:    %mulf_0_reg.in, %mulf_0_reg.write_en, %mulf_0_reg.clk, %mulf_0_reg.reset, %mulf_0_reg.out, %mulf_0_reg.done = calyx.register @mulf_0_reg : f32, i1, i1, i1, f32, i1
+// CHECK-DAG:    %std_ieee754.mul_0.clk, %std_ieee754.mul_0.reset, %std_ieee754.mul_0.go, %std_ieee754.mul_0.control, %std_ieee754.mul_0.left, %std_ieee754.mul_0.right, %std_ieee754.mul_0.roundingMode, %std_ieee754.mul_0.out, %std_ieee754.mul_0.exceptionalFlags, %std_ieee754.mul_0.done = calyx.std_ieee754.mul @std_ieee754.mul_0 : i1, i1, i1, i1, f32, f32, i3, f32, i5, i1
```

Comment:
looking good? is this what you are looking for? @rachitnigam @andrewb1999

Reply:
This looks right! Although, why are the names of the ports of the `std_ieee754`? Can we name the module mulFN so that the name is a little more readable?

Reply:
Can I double check if I understand you correctly?
In CIRCT-Calyx, we'd like to have:
```mlir
%mulFN_0.clk, %mulFN_0.reset, ... = calyx.std_ieee754.mul @mulFN_0 : i1, i1, ....
```
not:
```mlir
%std_ieee754.mul_0.clk, %std_ieee754.mul_0.reset, ... = calyx.std_ieee754.mul @std_ieee754.mul_0 : i1, i1, ....
```

And after `--export-calyx`, we'd like to have:
```mlir
import "primitives/float/mulFN.futil";
component() {
  cell {
    mulFN_0 = mulFN(8, 24, 32);
  }
  ...
}
```

Reply:
Yup! Mostly this is because `mulFN_0 = std_ieee754.mul_0(...)` is not valid Calyx because of the `.` in the name.